node/src/Unison/Runtime/Html.hs

```diff
@@ -0,0 +1,26 @@
+module Unison.Runtime.Html where
+
+import Data.Maybe (listToMaybe, catMaybes, mapMaybe)
+import Data.Text (Text, toLower, pack)
+import Text.HTML.TagSoup
```

Comment:
Can you explicitly import any operators you are using and/or import the module qualified? Most readers of this code are unlikely to be familiar with what tag soup provides